util/src/main/java/com/redhat/lightblue/util/metrics/DropwizardRequestMetrics.java

```diff
@@ -72,20 +69,25 @@ private static String errorNamespace(String metricNamespace, Throwable exception
         return e.getClass();
     }
 
-    private String checkNullVersion(String version) {
+    /**
+     * If version is null,replace with default.
+     * Also replace all . in version with _ as graphite treats . as standard separator and can cause issues in queries.
+     * 
+     */
+    private String checkVersion(String version) {
```

Comment:
How about `cleanVersion` or `formatVersion`? `checkVersion` implies it is doing some kind of validation and would expect it to throw an exception

Reply:
Fixed name with formatVersion.